core/geometry/src/Geometry.ts

```diff
@@ -7,6 +7,7 @@
  * @module CartesianGeometry
  */
 
+import { assert } from "@itwin/core-bentley";
```

Comment:
This doesn't seem to be used anywhere, which I would expect to generate either a lint error or a lint warning.

Reply:
No not yet. I'm making changes locally and replying to the comments so I dont forget. Please wait for the next commit. FYI, I changed the PR to draft for now.